util/util_test.go

```diff
@@ -6,6 +6,12 @@ import (
 	"testing"
 )
 
+func TestBinaryInstalled(t *testing.T) {
```

Comment:
true. thanks for that! you are right the test should run all the tests for real. i will add the osc one later!